### PR TITLE
Add same crash buttons in WPF puppet as Xamarin.Forms app

### PR DESCRIPTION
**Pull request by guperrot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the UWP implementation?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Add same crash buttons in WPF puppet as Xamarin.Forms app

## Related PRs or issues

[AB#62886](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/62886)

### CODE REVIEW

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs [hunk 1]
```diff
@@ -84,15 +91,112 @@ private void trackEvent_Click(object sender, RoutedEventArgs e)
...
+
+        private void crashesEnabled_Checked(object sender, RoutedEventArgs e)
+        {
+            if (crashesEnabled.IsChecked.HasValue)
+            {
+                Crashes.SetEnabledAsync(crashesEnabled.IsChecked.Value).Wait();
+            }
+        }
+
+        public int val
```
Comment (MatkovIvan) [inline Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: Naming

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs [hunk 2]
```diff
@@ -84,15 +91,112 @@ private void trackEvent_Click(object sender, RoutedEventArgs e)
...
+
+        private static class FakeService
+        {
+            public static async Task DoStuffInBackground()
+            {
+                await Task.Run(() => throw new IOException("Server did not respond"));
+            }
+        }
+
+        private void crashesEnabled_Checked(object sender, RoutedEventArgs e)
```
Comment (MatkovIvan) [inline Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: Naming

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs [hunk 3]
```diff
@@ -16,28 +19,31 @@ namespace Contoso.WPF.Puppet
...
-            { LogLevel.Verbose, AppCenterLog.Verbose },
-            { LogLevel.Debug, AppCenterLog.Debug },
-            { LogLevel.Info, AppCenterLog.Info },
-            { LogLevel.Warn, AppCenterLog.Warn },
-            { LogLevel.Error, AppCenterLog.Error }
-        };
+        private static readonly IDictionary<LogLevel, Action<string, string>> LogFunctions =
+            new Dictionary<LogLevel, Action<string, string>>
+            {
+                {LogLevel.Verbose, AppCenterLog.Verbose},
```
Comment (MatkovIvan) [inline Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: Is it formatted correctly?

Comment (guperrot) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: was auto formatted

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs (hunk 1 above)
Comment (guperrot) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: I didn't write this code, I wonder how it ended up in the PR, will look.

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs (hunk 2 above)
Comment (guperrot) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: I didn't write this code, I wonder how it ended up in the PR, will look.

Comment (guperrot) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: This is generated code.

Comment (guperrot) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: Aligned all existing fields to PascalCase

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs (hunk 3 above)
Comment (guperrot) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: ctrl k ctrl d does not update it.

**Pull request merged**